ui/summary.go

```diff
@@ -238,21 +249,22 @@ func NonTrendMetricValueForSum(t time.Duration, timeUnit string, m *stats.Metric
 	}
 }
 
-func DisplayNameForMetric(m *stats.Metric) string {
+func displayNameForMetric(m *stats.Metric) string {
 	if m.Sub.Parent != "" {
 		return "{ " + m.Sub.Suffix + " }"
 	}
 	return m.Name
 }
 
-func IndentForMetric(m *stats.Metric) string {
+func indentForMetric(m *stats.Metric) string {
 	if m.Sub.Parent != "" {
 		return "  "
 	}
 	return ""
 }
 
-func SummarizeMetrics(w io.Writer, indent string, t time.Duration, timeUnit string, metrics map[string]*stats.Metric) {
+func (s *Summary) summarizeMetrics(w io.Writer, indent string, t time.Duration,
```

Comment:
This is true, but it's only triggered now because of the signature change.

Can we ignore this for now, to avoid spending more time on this refactor?